src/reflect/scala/reflect/runtime/JavaUniverseForce.scala

```diff
@@ -3,7 +3,7 @@ package scala.reflect
 package runtime
 
 trait JavaUniverseForce { self: runtime.JavaUniverse  =>
-  def force() {
+  def force(): Unit = {
```

Comment:
There is a comment here that it's generated code. Maybe this should be fixed in the generator? I have no idea *where* this code is generated.

Reply:
It's not _actually_ generated; the contents of this method are "verified" by some `run` partest (on my phone, but the file name is `run/t1234-universe-codegen` for a sufficiently large value of 1234). So this is apparently fine.

Reply:
Then the comment probably needs to be removed, don't you think? Not sure if that's in scope of this PR under the boyscout rule (if @xuwei-k feels like including that) or that it should be a separate PR.